src/rculfhash.c

```diff
@@ -1603,6 +1648,20 @@ struct cds_lfht *_cds_lfht_new(unsigned long init_size,
 			const struct cds_lfht_mm_type *mm,
 			const struct rcu_flavor_struct *flavor,
 			pthread_attr_t *attr)
+{
+	return _cds_lfht_new_with_alloc(init_size,
+			min_nr_alloc_buckets, max_nr_buckets,
+			flags, mm, NULL, flavor, attr);
+}
```

Comment:
Please move the implementation of _cds_lfht_new() after the implementation of _cds_lfht_new_with_alloc.

Reply:
Moved.